modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/SnapshotUri.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.table.distributed.raft.snapshot;
+
+import java.util.UUID;
+
+/**
+ * Snapshot URI.
+ */
+public class SnapshotUri {
+    /** Length of UUID's sting representation in characters. Includes 32 hexadecimals and 4 dashes. */
+    private static final int UUID_LENGTH = 36;
+
+    /**
+     * Creates a string representation of the snapshot URI.
+     *
+     * @param snapshotId Snapshot id.
+     * @param nodeName Sender node name.
+     */
+    public static String toStringUri(UUID snapshotId, String nodeName) {
```

Comment:
Why creating an instance if you can create a string immediately?

Reply:
Because it looks better =) `Integer.toString` is a bad example, because Integer semantically equivalent to int and it only accepts a single parameter, which makes it clear to use. Nevertheless, it's your call, I won't insist

Reply:
I mean, `toStringUri(x, y)` is better than `new SnapshotUri(x, y).toString()`, literally the same example